requirements.txt

```diff
@@ -4,4 +4,5 @@ numpy==1.22.4
 python-multipart==0.0.5
 requests==2.27.1
 shapely==1.8.2
-uvicorn==0.17.6
\ No newline at end of file
+uvicorn==0.17.6
+xmltodict
```

Comment:
Is a version number needed for xmltodict?

Reply:
Updated